src/morphology_workflows/tasks/repair.py

```diff
@@ -256,6 +256,6 @@ class PlotSmoothDiameters(SkippableMixin(True), ElementValidationTask):
     def inputs(self):
         """ """
         return {
-            RepairNeurites: {"morph_path": "morph_path"},
+            FixZeroDiameters: {"morph_path": "morph_path"},
```

Comment:
Why changing this?

Reply:
so it's consistent with the SmoothDiameter task, this was a mistake from previous PR

Reply:
Ah ok